modules/apps/saved-content/saved-content-web/src/main/java/com/liferay/saved/content/web/internal/portlet/action/EditSavedContentMVCActionCommand.java

```diff
@@ -87,37 +83,31 @@ private JSONObject _updateSavedContentEntry(ActionRequest actionRequest) {
 					themeDisplay.getScopeGroupId(), userId, className, classPK,
 					ServiceContextFactory.getInstance(
 						SavedContentEntry.class.getName(), actionRequest));
-				jsonObject.put("saved", Boolean.TRUE);
 
-				return jsonObject;
+				return JSONUtil.put("saved", Boolean.TRUE);
 			}
 
 			_savedContentEntryService.deleteSavedContentEntry(
 				savedContentEntry);
 
-			jsonObject.put("saved", Boolean.FALSE);
+			return JSONUtil.put("saved", Boolean.FALSE);
 		}
 		catch (PortalException portalException) {
 			if (_log.isDebugEnabled()) {
 				_log.debug(portalException);
 			}
 
-			jsonObject.put(
+			return JSONUtil.put(
 				"errorMessage",
 				_language.get(
 					_portal.getHttpServletRequest(actionRequest),
 					"an-unexpected-error-occurred"));
```

Comment:
I would create a method to return this JSON. The same code can also be seen above.

Reply:
not be done because removed code.